src/main/java/com/unascribed/fabrication/Resolvable.java

```diff
@@ -91,8 +91,7 @@ public boolean equals(Object obj) {
 		if (getClass() != obj.getClass()) return false;
 		Resolvable<?> other = (Resolvable<?>) obj;
 		if (!Objects.equal(id, other.id)) return false;
-		if (registry != other.registry) return false;
-		return true;
+		return registry == other.registry;
```

Comment:
This is autogenerated and by modifying it won't be recognized if I make changes to this class later.